barretenberg/cpp/src/barretenberg/circuit_checker/translator_circuit_checker.hpp

```diff
@@ -0,0 +1,59 @@
+#pragma once
+#include "barretenberg/translator_vm/translator_circuit_builder.hpp"
+namespace bb {
+class TranslatorCircuitChecker {
```

Comment:
the code present in the header and source of this class has just simply been relocated from TranslatorCircuitBuilder with no modifications to actual functionality